Assets/Global/Scripts/Player/CheatCodeSystem.cs

```diff
@@ -0,0 +1,135 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+using System.Collections.Generic;
+using UnityEngine.SceneManagement;
+
+public class CheatCodeSystem : MonoBehaviour
+{
+    private PlayerInput playerInput;
+    public float maxComboTime = 2f; // Max time to complete the combo
+    private float comboTimer;
+    private string currentSequence = "";
+    private List<string> cheatCodes = new List<string> { "LULDR", "RLLRD", "UUDLR", "DDRLU", "UDLRRLDD"}; // Example sequences
+
+    private void Awake()
+    {
+        playerInput = GetComponent<PlayerInput>();
+    }
+
+    private void OnEnable()
```

Comment:
Tthis code isnt needed if you set it up through the PlayerInput-Component in the editor.
Would also be consistent with the other input handling.